poetry.lock

```diff
@@ -310,7 +318,7 @@ flake8 = ">=3.5,<5"
 
 [[package]]
 name = "flake8-bugbear"
-version = "22.7.1"
+version = "22.6.22"
```

Comment:
I think yo downgraded some packages by mistake. Try to revert the previous lockfile, do a `poetry install` and add the dev dependencies by `poetry add --dev <dep>` for each one. That should be cleaner

Reply:
Ok cool